src/StoreKeeper/WooCommerce/B2C/Imports/ProductImport.php

```diff
@@ -1033,29 +1040,57 @@ protected function updateProductMeta($newProduct, Dot $dotObject, array $log_dat
         $this->debug('storekeeper_id added to post.', $log_data);
 
         if ($dotObject->has('flat_product.content_vars')) {
+
             $nonAttributeOptions = array_filter(
                 $dotObject->get('flat_product.content_vars'),
                 function ($attribute) {
                     return !key_exists('attribute_option_id', $attribute);
                 }
             );
-            foreach ($nonAttributeOptions as $attribute) {
+
+            // Find attributes that have 'attribute_option_color_hex'
+            $colorAttributes = array_filter(
```

Comment:
there also should be changes in `src/StoreKeeper/WooCommerce/B2C/Imports/AttributeOptionImport.php::processItem`